src/components/BooksForm.js

```diff
@@ -0,0 +1,34 @@
+const { v4: UuidV4 } = require('uuid');
```

Comment:
- [ ] **(OPTIONAL)** I advise you to put this module when you are creating the redux store since it only runs once, and then the Ids will be the same through the entire app. If you do it here, the code in UUID will be run every time this component renders.